static/app/views/releases/detail/overview/releaseAdoption.tsx

```diff
@@ -198,67 +208,121 @@ function ReleaseComparisonChart({
     allowEmptyPeriod: true,
   });
 
+  const isMobileProject = isProjectMobileForReleases(project.platform);
+  const adoptionStage = release.adoptionStages?.[project.slug].stage;
+  const adoptionStageLabel =
+    Boolean(adoptionStage) && ADOPTION_STAGE_LABELS[adoptionStage];
+  const allEnvironments = environment.length === 0;
+  const multipleEnvironments = environment.length > 1;
+
   return (
-    <RelativeBox>
-      <ChartLabel top="0px">
-        <ChartTitle>
-          {t('Sessions Adopted')}
-          <QuestionTooltip
-            position="top"
-            title={t(
-              'Adoption compares the sessions of a release with the total sessions for this project.'
-            )}
-            size="sm"
+    <Wrapper>
+      {isMobileProject && (
+        <Feature features={['release-adoption-stage']}>
+          <SidebarSectionTitle title={t('Adoption Stage')} />
+          {adoptionStageLabel && !allEnvironments && !multipleEnvironments ? (
+            <StyledTooltip title={adoptionStageLabel.tooltipTitle}>
```

Comment:
We should update `tooltipTitle` texts according to the latest designs:
https://www.figma.com/file/jsZJ9v8ditpm4uBISafH9P/Releases-V4?node-id=1241%3A16632